packages/react-composites/src/composites/CallComposite/pages/LobbyPage.tsx

```diff
@@ -71,14 +71,21 @@ export const LobbyPage = (props: LobbyPageProps): JSX.Element => {
 const overlayProps = (
   strings: CallCompositeStrings,
   inLobby: boolean,
-  remoteParticipants?: RemoteParticipantState[]
+  remoteParticipants: RemoteParticipantState[]
 ): LobbyOverlayProps => {
   /**
    * Only grab the first participant because there will only be one in this situation.
    * when starting a call with multiple people the call goes to the connected state and composite goes directly to
    * videoGallery.
+   *
+   * We also need to check the participant state since in a group call the remote participants array will populate just before
+   * the user joins.
    */
-  const outboundCallParticipant = remoteParticipants ? remoteParticipants[0] : undefined;
+  const outboundCallParticipant: RemoteParticipantState =
+    remoteParticipants[0] && ['Ringing', 'Connecting'].includes(remoteParticipants[0].state)
+      ? remoteParticipants[0]
+      : undefined;
+  outboundCallParticipant && console.log(outboundCallParticipant.state);
```

Comment:
Is this PR missing a commit? It tries grab a outboundCallParticipant then just console logs it currently